Troubleshooting: add an article about gray boxes in grids (#5131)
DevExpress/devextreme-documentation: concepts/80 Troubleshooting/05 DataGrid and TreeList Issues/10 Gray Boxes Appear After Data Loading/10 Use Another Machine for Testing.md

> If DataGrid or TreeList is complex or the Internet connection is bad, it may look like gray boxes are displayed permanently, but in fact the rows take a long time to load. Replace your data source with local data and open your project on a different machine.

Reviewer: ```suggestion
If DataGrid or TreeList is complex or the Internet connection is slow or unstable, gray boxes can be displayed for a very long time. If gray boxes take too long to disappear, you need to replace a remote data source with a local data source and open your project on a different machine.
```

> If DataGrid or TreeList is complex or the Internet connection is slow or unstable, gray boxes can be displayed for a very long time. If gray boxes take too long to disappear, you need to replace a remote data source with a local data source and open your project on a different machine.

Reviewer: Well, I don't believe it's wise to address components as "complex". We speak of content, actually.

```suggestion
If DataGrid or TreeList content is complex or the Internet connection is slow or unstable, gray boxes can be displayed for a very long time. If gray boxes take too long to disappear, you need to replace a remote data source with a local data source and open your project on a different machine.
```

> If DataGrid or TreeList content is complex or the Internet connection is slow or unstable, gray boxes can be displayed for a very long time. If gray boxes take too long to disappear, you need to replace a remote data source with a local data source and open your project on a different machine.

Reviewer 2: ```suggestion
If DataGrid or TreeList content is complex or the Internet connection is slow or unstable, gray boxes may be displayed for a longer time. If gray boxes take too long to disappear, you need to replace a remote data source with a local data source and open your project on a different machine.
```
